src/slice.h

```diff
@@ -4,9 +4,18 @@
 extern SEXP syms_vec_slice_dispatch_integer64;
 extern SEXP fns_vec_slice_dispatch_integer64;
 
+enum vctrs_materialize {
```

Comment:
I think we started out with british spelling in this repo.

Reply:
We actually use the Z in quite a few places:

- ALTREP rle inspect method `"vctrs_altrep_rle (len=%d, materialized=%s)\n"`
- `altrep_rle_string_Materialize()`
- `compact_materialize()`

I think I'm just going to leave it for now and we can change them all over later to be consistent